.azure-pipelines/util/build-steps.yml

```diff
@@ -4,7 +4,9 @@ parameters:
   powerShellPlatform: ''
 
 steps:
-- template: get-github-pat-steps.yml
+- checkout: self
```

Comment:
How long does this extra checkout step take to execute?

Reply:
It's not an extra step. If we do not add it manually, Azure Pipelines will checkout "self" repository anyway. We only add it for adding a parameter (by default it's false).